src/accelerate/data_loader.py

```diff
@@ -1182,7 +1182,8 @@ def __len__(self):
 
 class SkipDataLoader(DataLoaderAdapter, DataLoaderStateMixin):
     """
-    Subclass of a PyTorch `DataLoader` that will skip the first batches.
+    Subclass of a PyTorch `DataLoader` that will skip the first batches. Generally it's preferable to use
+    `skip_first_batches`/`torchdata.StatefulDataLoader` instead of this class.
```

Comment:
in the code below, we are still exposing the `use_stateful_dataloader` arg with `SkipDataLoader`. Should we remove that to not cause confusion ?